### PR TITLE
duplicate bullet point

### PR DESCRIPTION
**Pull request by JoanneOFlynn2018**

### CODE REVIEW

File: doc-Service-Telemetry-Framework/modules/proc_creating-openstack-environment-file-for-multiple-clouds.adoc [hunk 1]
```diff
@@ -39,7 +39,6 @@ endif::include_when_13[]
 * You have retrieved the {MessageBus} route address. For more information, see xref:retrieving-the-qdr-route-address_assembly-completing-the-stf-configuration[].
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_creating-openstack-environment-file-for-multiple-clouds.adoc:39]:
```suggestion
* You have created your list of clouds objects. For more information about creating the content for the clouds parameter, see https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/service_telemetry_framework_1.3/assembly-installing-the-core-components-of-stf_assembly#clouds_assembly-installing-the-core-components-of-stf[clouds].

* You have retrieved the {MessageBus} route address. For more information, see xref:retrieving-the-qdr-route-address_assembly-completing-the-stf-configuration[].
```

Comment (JoanneOFlynn2018) [reply doc-Service-Telemetry-Framework/modules/proc_creating-openstack-environment-file-for-multiple-clouds.adoc:39]: This change is linked to this DDF BZ: 
https://bugzilla.redhat.com/show_bug.cgi?id=1980060

Comment (leifmadsen) [reply doc-Service-Telemetry-Framework/modules/proc_creating-openstack-environment-file-for-multiple-clouds.adoc:39]: I'm concerned about the link not being very dynamic. Maybe the `_1.3` can be replaced with some other parameter we specify? Shouldn't we be able to use an `xref` here or something?

Comment (JoanneOFlynn2018) [reply doc-Service-Telemetry-Framework/modules/proc_creating-openstack-environment-file-for-multiple-clouds.adoc:39]: Not sure why I didn't use an xref first day. Anyway, fixed it up just now :)

**Review by leifmadsen — APPROVED** [review]

**Comment by leifmadsen** [comment]
Does this also need to land on `main` or is that already completed? Thanks!

**Pull request merged**